ONSdigital/eq-terraform
# EQ-24 Removing smoke test from terraform run

**What**
Removing the smoke test from terraform. It will now be done via Jenkins.

**How to test**
Checkout this branch and run a terraform build. The smoke tests should not be run

**Who can review**
Anyone but @warrenbailey
